eng/Packages.Data.props

```diff
@@ -103,6 +103,12 @@
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
     <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
     <PackageReference Update="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration" Version="8.0.0" />
```

Comment:
We'll need to figure out if this still works if we have to downgrade these and/or if we'll need to polyfill to support target scenarios.